main.py

```diff
@@ -21,7 +21,7 @@
             lifestyle.general.age(input('DOB (DD-MM-YYYY): '))
         elif cmd == 'uc':
             ankii_math.conversions.uc()
-        elif cmd == 'bin':
+        elif cmd == 'hex':
```

Comment:
bin wasn't suitable to this command as it was used to show byte patterns in hex.

Reply:
Gutka thuk k baad kr

Reply:
Please consider talking in a professional manner this is my official GitHub Mr. Ankit